tests/core/pyspec/eth2spec/test/helpers/execution_payload.py

```diff
@@ -1,4 +1,4 @@
-def build_empty_execution_payload(spec, state):
+def build_empty_execution_payload(spec, state, randao_mix):
```

Comment:
This could be an optional parameter, to avoid the `, spec.Bytes32()` arguments everywhere

Reply:
I was thinking about that. But my gut is against default arguments in testing helpers, we may have a separate function instead, like `build_empty_execution_payload_with_randao`, wdyt?